src/aind_metadata_service/labtracks/client.py

```diff
@@ -125,12 +125,15 @@ def get_subject_info(self, subject_id: str) -> ModelResponse:
         Parameters
         ----------
         subject_id: str
+            This is the id in the LabTracks ANIMALS_COMMON table
         """
+        if not subject_id.isalnum():
```

Comment:
Do we want to check for anything else in `subject_id` (e.g. int, length)?

Reply:
Labtracks subject_ids should only be integers so maybe do an int check instead of alphanumeric.

Reply:
Both `isalnum()` and `isnumeric()` return False on empty strings, so min length of 1 is automatically checked already.

Reply:
Sounds good. We probably don't need an additional check.